packages/design-system/src/connect/components/home-screen/home-screen.tsx

```diff
@@ -0,0 +1,31 @@
+import { Icon } from "@/powerhouse";
+import { ComponentPropsWithRef, ForwardedRef, forwardRef } from "react";
+import { twMerge } from "tailwind-merge";
+import { HomeScreenItem } from "../home-screen-item";
+type InputProps = ComponentPropsWithRef<"input">;
+type HomeScreenProps = Omit<InputProps, "className"> & {
+  readonly children: React.ReactNode;
+  readonly containerClassName?: string;
+};
+export const HomeScreen = forwardRef(function HomeScreen(
```

Comment:
I don't think we need to use forwardRef on all components. Only when there is a need for it.

Reply:
Generally you should only forwardRef when you see an actual error that results from not forwarding a ref.